.jenkins/lsu-perftests/batch.sh

```
@@ -16,9 +16,14 @@ status_computation_and_artifacts_storage() {
 
     # Copy the testing directory for saving as an artifact
     cp -r ${build_dir}/Testing ${src_dir}/${configuration_name}-Testing
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)